src/text/font.rs

```diff
@@ -128,20 +124,20 @@ impl Font {
         };
 
         Ok(Self {
-            data: data.to_owned(),
-            owned_ttf_font,
+            data: data.clone(),
+            face_index,
             units_per_em,
             metrics,
             glyphs: Default::default(),
         })
     }
 
     pub fn data(&self) -> &[u8] {
-        self.data.as_ref()
+        self.data.as_ref().as_ref()
     }
 
-    fn font_ref(&self) -> &TtfFont<'_> {
-        self.owned_ttf_font.as_face_ref()
+    fn font_ref(&self) -> TtfFont<'_> {
```

Comment:
Note to self: fix the call side to not call this for each uncached glyph but maybe as a with_face helper and a closure only once.

Reply:
Done :-) (using https://crates.io/crates/ouroboros)